sdk/tables/Microsoft.Azure.WebJobs.Extensions.Tables/tests/TablesLiveTestBase.cs

```diff
@@ -41,8 +41,6 @@ protected TablesLiveTestBase(bool isAsync, bool useCosmos, bool createTable = tr
         {
             UseCosmos = useCosmos;
             _createTable = createTable;
-            // https://github.com/Azure/azure-sdk-tools/issues/2448
-            Sanitizer.BodyRegexSanitizers.Add(new BodyRegexSanitizer("(batch|changeset)_[\\w\\d-]+", "multipart_boundary"));
```

Comment:
This is no longer needed as we are using the internal SetBatchGuids method to ensure that the values are consistent between recording and playback.